### PR TITLE
Improves ambience

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
The changelog covers most of it. Basically ambience plays more consistently, *slightly* more often, and respects environmental effects like reverb. 

## Why It's Good For The Game
Muh immershun

## Changelog
:cl: Kapu1178, Baystation12
sound: Ambience is now sourced from the environment instead of sent directly to the client.
sound: Space no longer plays station humming,  and Lavaland is more lava-y.
sound: APCs changing powerstate is accompanied by a breaker switch sound.
sound: Maintenance now has its own droning loop, as well as more random sounds that play more often.
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/ambience.dm [hunk 1]
```diff
@@ -18,32 +19,73 @@ SUBSYSTEM_DEF(ambience)
...
 
-	ambience_listening_clients[to_process] = world.time + rand(current_area.min_ambience_cooldown, current_area.max_ambience_cooldown)
+	return rand(min_ambience_cooldown, max_ambience_cooldown) + (new_sound.len * 10) //Convert to deciseconds
 
 /datum/controller/subsystem/ambience/proc/remove_ambience_client(client/to_remove)
 	ambience_listening_clients -= to_remove
+	client_old_areas -= to_remove
 	currentrun -= to_remove
+
+/area/maintenance
```
Comment (JohnFulpWillard) [inline code/controllers/subsystem/ambience.dm:70]: this area doesn't exist

Comment (Kapu1178) [reply code/controllers/subsystem/ambience.dm:70]: cock

**Comment by Acer202** [comment]
Silent space? No more dilbert tunes to vibe to?

**Review by Mothblocks — CHANGES_REQUESTED** [review]
> Deaf mobs will no longer hear ambience.

I think this is a bad change and would hurt game feel a lot for the sake of realism. Currently walking around somewhere like medbay while deaf is actually pretty thematic, like a subtle hum.

**Comment by Kapu1178** [comment]
random sql failure rerun pls

**Comment by Shadowflame909** [comment]
oh gosh what an important PR that went overlooked by the majority of the playerbase

The new sounds are very startling and sounds almost like you got some sort of new fangled antag. I most compare it to changeling. Distracting and not at all ambience-like. More like OST!

Very sad!

**Pull request merged**